vpr/src/route/rr_graph.cpp

```diff
@@ -2568,7 +2565,7 @@ static void build_unidir_rr_opins(RRGraphBuilder& rr_graph_builder,
         }
 
         /* Add in direct connections */
-        get_opin_direct_connections(rr_graph_builder, i, j, side, pin_index, opin_node_index, rr_edges_to_create, rr_nodes,
```

Comment:
Since the function has no use of ``rr_nodes``, it is time to put ``rr_graph_view`` as one of the argument. I suggest to put it right after the ``rr_graph_builder`` in the argument list.